codejail/safe_exec.py

```diff
@@ -36,6 +37,12 @@ def safe_exec(code, globals_dict, files=None, python_path=None, slug=None,
     >>> jail = codejail.jail.get_codejail("python")
     >>> jail.safe_exec(...)
     """
+    if not is_configured("python"):
+        venv = sibling_sandbox_venv()
+        if not venv:
+            raise Exception("Couldn't find sandbox virtualenv")
+        configure("python", os.path.join(venv, "bin/python"), user="sandbox", lang=languages.python2)
```

Comment:
I would love for you to help figure out how to do that.  Keep in mind that the tests need pass, both with and with sandboxes in the environment.  We have Django middleware that configures codejail, but that middleware isn't run during the safe_exec tests, since they are not in a Django app.